aptos-move/block-executor/src/value_exchange.rs

```diff
@@ -0,0 +1,216 @@
+// Copyright © Aptos Foundation
```

Comment:
you'll need to rebase and update this file, all these functions changed.

Reply:
That was great because rebase would miss it and it did.
I copied the latest for TemporaryValueToIdentifierMapping, other utilities were new and should be okay.